playground/react/.env

```diff
@@ -1,11 +1,7 @@
-# Flow: PasskeySignupWithEmailOTPFallback
-REACT_APP_CORBADO_PROJECT_ID=pro-503401103218055321
+# REACT_APP_CORBADO_PROJECT_ID_PasskeyWithEmailOtpFallback=pro-503401103218055321
+# REACT_APP_CORBADO_PROJECT_ID_EmailOtpWithPasskeyAppend=pro-8793272752372175738
+# REACT_APP_CORBADO_PROJECT_ID_PasskeyWithEmailLinkFallback=pro-423122463392265807
+# REACT_APP_CORBADO_PROJECT_ID_EmailLinkWithPasskeyAppend=pro-3962864614985263396
 
-# Flow: EmailOTPSignupWithPasskey
-# REACT_APP_CORBADO_PROJECT_ID=pro-8793272752372175738
-
-# Flow: PasskeySignupWithEmailOTPFallback + Email Link verification method
-# REACT_APP_CORBADO_PROJECT_ID=pro-423122463392265807
-
-# Flow: EmailOTPSignupWithPasskey + Email Link verification method
-# REACT_APP_CORBADO_PROJECT_ID=pro-3962864614985263396
+REACT_APP_CORBADO_PROJECT_ID_EmailOtp=pro-503401103218055321
```

Comment:
We need to adapt .env.production as well (otherwise the vercel deployment and the E2E tests will fail). I have mentioned the two projects that should be used in the GH issue).